src/main/java/com/zaxxer/sansorm/SqlClosure.java

```diff
@@ -123,7 +122,7 @@ public static void setDefaultDataSource(final DataSource ds)
     * @return the result specified by the lambda
     * @since 2.5
     */
-   public static final <V> V sqlExecute(final SqlFunction<V> functional)
+   public static <V> V sqlExecute(final SqlFunction<V> functional)
```

Comment:
statics are always final

Reply:
statics are not always final.  try it.  😁

Reply:
do you mean it can be hidden? :) ok, so what should i do? leave it or revert it?

Reply:
Sorry, not enough sleep.  I glanced at that and thought class member, not class method.

Quite right, unnecessary.